apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java

```diff
@@ -273,6 +273,27 @@
             public static int SQL_PARAMETERS_MAX_LENGTH = 512;
         }
 
+        public static class POSTGRESQL {
+            /**
+             * If set to true, the parameters of the sql (typically {@link java.sql.PreparedStatement}) would be
+             * collected.
+             */
+            public static boolean TRACE_SQL_PARAMETERS = true;
+
+            /**
+             * For the sake of performance, limit the max number of parameter
+             */
+            public static int SQL_PARAMETERS_MAX_COUNT = 20;
+
+            /**
+             * For the sake of performance, SkyWalking won't save the entire parameters string into the tag, but only
+             * the first {@code SQL_PARAMETERS_MAX_LENGTH} characters.
+             *
+             * Set a negative number to save the complete parameter string to the tag.
+             */
+            public static int SQL_PARAMETERS_MAX_LENGTH = 512;
```

Comment:
Add these configs to documentations, otherwise, nobody can find this feature until they read the source codes